zerver/lib/outgoing_webhook.py

```diff
@@ -145,7 +146,7 @@ def send_response_message(bot_id, message, response_message_content):
         recipients = [recipient['email'] for recipient in message['display_recipient']]
         if bot_user.email in recipients:
```

Comment:
@timabbott it's copying `bot_user.email` because of this line; `send_response_message` requires a private message to be sent to the bot itself as well.

Reply:
I see.  I think that's because this code is actually totally broken; we should definitely not be passing `forwarded_user_profile` at all here, and I don't think that check servers a purpose either.

Reply:
Hmm, [this](https://github.com/zulip/zulip/commit/53a8b2ac87bfe2f4bc7221eefbabfce6d471fec3) is the respecitve commit, but it doesn't seem to belong to a PR?